coop_badge_reader/models/res_users.py

```diff
@@ -1,16 +1,17 @@
-# -*- coding: utf-8 -*-
 # Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
 # @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# Copyright (C) 2020-Today: Druidoo (<https://www.druidoo.io>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models
+from odoo import models, api
 
 
 class ResUsers(models.Model):
-    _inherit = 'res.users'
+    _inherit = "res.users"
 
     # TODO Improve Odoo-JS lib
     # Exist because 'has_group' function doesn't accept context args
     # that is not manage by the Odoo-JS lib
-    def check_group(self, cr, uid, group_ext_id, context=None):
-        return self.has_group(cr, uid, group_ext_id)
+    @api.model
+    def check_groups(self, group):
```

Comment:
This function name changed from `check_group` to `check_groups`.
But I also see that you changed the rpc method in the javascript file to call `user_has_groups` instead, which I think it's much better actually.

Since we're not using it, can we remove this ? Looks like we can remove this file entirely.
Make sure it's not used anywhere else, though..